lib/formtools/plugins/plugins-helpers.js

```diff
@@ -226,6 +226,41 @@ var overviewDefaults = function (overview, labels) {
 
     overview.summary.label = overview.summary.label || 'Summary';
 
+
+    //overview Details
+    if (!overview.details || typeof overview.details !== 'object') {
+        // defaults to {}
```

Comment:
This comment isn't helpful. It can be removed. I prefer to keep comments to provide information that isn't obvious from reading the code. Could you please also ensure comments are in the format `// comment here` (noting the space between `//` and the first part of the comment).